control-station/src/services/PodSocketClient.ts

```diff
@@ -123,8 +124,6 @@ class PodSocketClient {
 
 	private onConnect(): void {
 		console.log("Connected to server as", this.socket.id);
-		// TODO: On connecting, the state below should be what's provided by the pod
-		// if it's already running. Otherwise, the states should be State.Init
```

Comment:
Question: is there a particular reason this comment was removed?

Reply:
I might have accidentally removed it while changing files